<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormFiller</class>
 <widget class="QWidget" name="FormFiller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Générateur de champ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/res.qrc">
    <normaloff>:/gear.png</normaloff>:/gear.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <widget class="QTreeView" name="actExplorerTV">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTableWidget" name="FormsTW">
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Clé</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Valeur</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="OtherGenerationTB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Générer, sauvegarder et copier dans le presse papier</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::MenuButtonPopup</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="CharacterCB"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
